libs/domain/picking/src/components/user-profile/user-profile.component.ts

```diff
@@ -32,16 +31,17 @@ export class UserProfileComponent extends LitElement {
 
   protected route = signal(this.routerService.route());
   protected pendingSyncs = signal(
-    this.syncSchedulerService
+    resolve(SyncSchedulerService)
       .hasPending()
       .pipe(tap((pending) => (this.hasPendingSyncs = pending)))
```

Comment:
Btw why do you need a side effect that sets `this.hasPendingSyncs` property if you can just use `this.pendingSyncs` signal?